edx_sysadmin/models.py

```diff
@@ -1,23 +1,19 @@
 """
 Database models for edx_sysadmin.
 """
+from django.db import models
+from jsonfield.fields import JSONField
 
-import mongoengine
+from opaque_keys.edx.django.models import CourseKeyField
 
-from xmodule.modulestore.mongoengine_fields import CourseKeyField
 
+class CourseGitLog(models.Model):
+    """CourseGitLog to store git-logs of courses imported from github"""
 
-class CourseImportLog(mongoengine.Document):
-    """Mongoengine model for git log"""
-
-    course_id = CourseKeyField(max_length=128)
-    # NOTE: this location is not a Location object but a pathname
-    location = mongoengine.StringField(max_length=168)
-    import_log = mongoengine.StringField(max_length=20 * 65535)
-    git_log = mongoengine.StringField(max_length=65535)
-    repo_dir = mongoengine.StringField(max_length=128)
-    commit = mongoengine.StringField(max_length=40, null=True)
-    author = mongoengine.StringField(max_length=500, null=True)
-    date = mongoengine.DateTimeField()
-    created = mongoengine.DateTimeField()
-    meta = {"indexes": ["course_id", "created"], "allow_inheritance": False}
+    course_id = CourseKeyField(max_length=255, db_index=True)
+    import_log = JSONField(null=True, blank=True)
```

Comment:
what is the difference between `git_log` and and `import_log`?

Reply:
These two names were used in the old mongo schema where `import_log` stores the `logs produced while importing the course into edx` (now I have changed its name to `course_import_log`). On the other hand, `git_log` stores the git details like `Cloning into 'demo-test-course'... Commit ID: e8458e3631c8b9332fdf9cda2bb46b7c464ad50a Branch: master` (now I have changed its name to `git_details`)

Reply:
@HamzaIbnFarooq `git_log` seems more relevant can we revert it?

Reply:
reverted back.